lib/exq_ui_web/live/components/table_component.html.heex

```diff
@@ -45,8 +47,19 @@
                     class: "form-check-input js-select-row"
                   ) %>
                 </td>
-                <%= for %{accessor: accessor} = column <- @columns do %>
-                  <td class={column[:text_break] && "text-break"}><%= accessor.(item) %></td>
+                <%= for column <- @columns do %>
+                  <td class={column[:text_break] && "text-break"}>
+                    <%= if column[:link] do %>
```

Comment:
Similar to the nav_link approach before, to cator for the least amount of changes, I allowed colums to provide an optional field :link to determind whether it is one. 
There are other more invasive/workflow changing ones, I felt that this way was the least disruptive. I'm happy to consider the other approaches